bootstrap.php

```diff
@@ -1,8 +1,20 @@
 <?php
+/**
+ * Event triggered before cache is written
+ */
+Event::register_function('front.display', function (&$html) {
+    $config = \Config::load('novius_tarteaucitron::config', false, true);
+    $code = '<script type="text/javascript">tarteaucitron.lang = '. json_encode(\Arr::get($config, 'translations', [])) .'</script>';
```

Comment:
Can’t we just put this in `novius_tarteaucitron/views/script.php` instead? It would be cleaner and shorter. Besides, this could trigger an error because `front.display` and `front.pageFound` are two different events